### ISSUE TITLE
Release 3.0.0 armhf rpm package thinks its x86_64

### ISSUE THREAD

**Issue by michaellindman** [clangen/musikcube#605]
The spec for the rpm build package denotes the architecture for the armhf file is x86_64 and rpm fails to install it due to thinking it is an incompaltible architecture when it really isn't.

```
[root@05d7ffa20bb9 tmp]# rpm -q --info musikcube_linux_3.0.0_armhf.rpm
Name        : musikcube
Version     : 3.0.0
Release     : 1
Architecture: x86_64
Install Date: (not installed)
Group       : unknown
Size        : 24309869
License     : BSD-3-Clause
Signature   : (none)
Source RPM  : musikcube-3.0.0-1.src.rpm
Build Date  : Thu Apr 20 19:05:23 2023
Build Host  : f3ff00778f74
Relocations : /usr
Vendor      : Humanity
URL         : https://www.musikcube.com
Summary     : musikcube built using CMake
Description :
DESCRIPTION
===========

This is an installer created using CPack (https://cmake.org). No additional installation instructions provided.
```

**Comment by clangen** [comment]
Thanks for the report, I'll try to get this fixed in the 3.0.1 build that I'm planning to release soon.

For (mostly my) reference, I believe the problem may be here: https://github.com/clangen/musikcube/blob/master/.cmake/GeneratePackage.cmake#L62 ... we are attempting to set the RPM's architecture, but I think the comma in the argument list may be messing things up. 

We should also rename `DEB_ARCHITECTURE` to `PACKAGE_ARCHITECTURE`, or something along those lines.

**Comment by clangen** [comment]
Apologies for the delay, but 3.0.1 was just released. The new `rpm` packages should have the correct architecture now, but I'm unable to test completely because I don't have an `armhf` environment that uses `rpm` packages. Please re-open if you are still seeing issues.
